internal/sbi/producer/callback.go

```diff
@@ -153,18 +153,37 @@ func AmPolicyControlUpdateNotifyUpdateProcedure(polAssoID string,
 				}
 			}()
 
+			configurationUpdateCommandFlags := &gmm_message.ConfigurationUpdateCommandFlags{
+				NeedGUTI:            true,
+				NeedAllowedNSSAI:    true,
+				NeedConfiguredNSSAI: true,
+				NeedRejectNSSAI:     true,
+				NeedTaiList:         true,
+				NeedNITZ:            true,
+				NeedLadnInformation: true,
+			}
+
 			// UE is CM-Connected State
 			if ue.CmConnect(models.AccessType__3_GPP_ACCESS) {
-				gmm_message.SendConfigurationUpdateCommand(ue, models.AccessType__3_GPP_ACCESS, nil)
+				gmm_message.SendConfigurationUpdateCommand(ue,
+					models.AccessType__3_GPP_ACCESS,
+					configurationUpdateCommandFlags,
+				)
+
 				// UE is CM-IDLE => paging
 			} else {
-				message, err := gmm_message.BuildConfigurationUpdateCommand(ue, models.AccessType__3_GPP_ACCESS, nil)
+				// TODO: Verify if need to start the timer
```

Comment:
Is this possible to be done in this PR by storing the configurationUpdateCommandFlags in the ue instead of ConfigurationUpdateMessage, and then rebuild (which has retransmit judjement this time) and call SendConfigurationUpdateCommand with the stored flag after the ue returns?

Reply:
> Is this possible to be done in this PR by storing the configurationUpdateCommandFlags in the ue instead of ConfigurationUpdateMessage, and call SendConfigurationUpdateCommand with the stored flag after the ue returns?

Yes, It is possible to use the configurationUpdateCommandFlags to replace ConfigurationUpdateMessage.
However, I think the timing of invoking SendConfigurationUpdateCommand should be deferred to the UE complete service request (i.e. re-establish the NAS signaling connection)